locales/jp.yml

```diff
@@ -38,3 +38,7 @@ jp:
     title: "お問い合わせ"
     message: "パートナーになりたい方々のご連絡をお待ちしております！"
     send: "送信"
+    firstname: "姓" #b/c switching order for Japanese
```

Comment:
this will actually create some issues, as the user will type their last name, but it will be saved as first name by the form...
ideally we would add a condition with ruby to this and other lines: https://github.com/odaiba-app/landing-page/pull/28/files#diff-8c04fcfcc6b376f8c794fe733bb889ba1ec20db5e66d2e011536136a167e0b87R231 
saying that when the langauge is english display the form as first name - last name, and when it's japanese display last name - first name
you can either try to implement the ruby logic or for now do just the correct translation and then we can have @alex-504 look at the ruby logic

Reply:
I'll just switch the Japanese name order for the correct correspondence, since I don't know ruby.